packages/sync/src/modules/class-themes.php

```diff
@@ -822,4 +841,16 @@ private function get_delete_theme_call() {
 	private function is_theme_switch() {
 		return did_action( 'after_switch_theme' );
 	}
+
+	/**
+	 * Return Total number of objects.
+	 *
+	 * @param array $config Full Sync config.
+	 *
+	 * @return int total
+	 */
+	public function total( $config ) {
```

Comment:
Seems like we define this in a bunch of module classes. Should we make it a trait?

Reply:
They are all not the same. Maybe one day 😅

Reply:
I mean, making a trait and having all the classes that have the total of 1 using that trait. Does that make sense?

Reply:
I want to change this numbers so for example in Options module I would love to change it to the number of options we are sending, will do it soonl

Reply:
Gotcha! 👍

Reply:
https://github.com/Automattic/jetpack/pull/13963/commits/a5c3143e11950047ea16ac0ab1a53a51bb253e98